### DISCUSSION TITLE
branch cleanup

### DISCUSSION THREAD

**Discussion by josephdadams** [josephdadams/TallyArbiter#722] [General]
@Jocke4f @MatteoGheza @hrueger  I'd like to clean up some branches that are stale/not needed. Anything you guys know right off that can go?

**Comment by hrueger** [comment]
I think the only two branches I have remaining here are [i18n](https://github.com/josephdadams/TallyArbiter/tree/i18n) (#240) and [tour](https://github.com/josephdadams/TallyArbiter/tree/tour) (#190)
However, since we're probably not going to finish either of them, feel free to cleanup. If that were to be implemented, a new start would probably be better anyway ;-)
